sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubMessage.java

```diff
@@ -66,4 +67,17 @@ public String getAttribute(String attribute) {
   public String getMessageId() {
     return messageId;
   }
+
+  @Override
+  public String toString() {
```

Comment:
`MoreObjects.toStringHelper()` provides a nice builder for a very predictable format in cases like this.

Reply:
Done.